ddtrace/constants.py

```diff
@@ -58,3 +58,16 @@
 AUTO_KEEP = 1
 # Use this to explicitly inform the backend that a trace should be kept and stored.
 USER_KEEP = 2
+
+# Used to check when a header can have an IP (in this order)
+IP_HEADERS = (
```

Comment:
Does this have to be a public constant? Can it be made internal?

Reply:
Done.